lib/smart_answer_flows/benefit-cap-calculator.rb

```diff
@@ -109,9 +109,9 @@ def define
 
       #Q6
       multiple_choice :single_couple_lone_parent? do
-        option :single
-        option :couple
-        option :parent
+        query.rates.keys.each do |rate|
+          option rate
+        end
```

Comment:
This was missed. The description text for single, couple, parent has been moved from the govspeak template and into the YAML file in a later commit.